<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminStudentAction</class>
 <widget class="QDialog" name="adminStudentAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:light gray</string>
  </property>
  <widget class="QLabel" name="removeTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>27</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Delete Student From System</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameTitle">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Student Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="idTitle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Student ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEntry">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="iDEntry">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="removeStudentButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Student</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="studentTable">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>371</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Academy Engraved LET</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="removeTitle_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>27</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Student Information Table</string>
   </property>
  </widget>
  <widget class="QLabel" name="removeTitle_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>User Modification</string>
   </property>
  </widget>
  <widget class="QLabel" name="studentConfirmedMessage">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
